src/main/java/com/catcher/batch/core/dto/MovieApiResponse.java

```diff
@@ -1,42 +1,105 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
+@CatcherJson(path = "")
 public class MovieApiResponse {
 
-    @JsonProperty("dailyBoxOfficeList")
-    private List<MovieItemDTO> response;
+    @JsonProperty("dates")
+    private MovieNowPlayingDate movieNowPlayingDate;
+
+    @Getter
+    private static class MovieNowPlayingDate {
+        @JsonProperty("maximum")
+        private String maximumDate;
+
+        @JsonProperty("minimum")
+        private String minimumDate;
+    }
+
+    @JsonProperty("total_pages")
+    @Getter
+    private Integer totalPages;
+
+    @JsonProperty("total_results")
+    @Getter
+    private Integer totalResults;
+
+    @JsonProperty("results")
+    private List<MovieItem> items;
 
+    @JsonIgnoreProperties(ignoreUnknown = true)
     @Getter
-    public static class MovieItemDTO {
-        private String rnum;
-        private String rank;
-        private String rankInten;
-        private String rankOldAndNew;
-        private String movieCd;
-        private String movieNm;
-        private String openDt;
-        private String salesAmt;
-        private String salesShare;
-        private String salesInten;
-        private String salesChange;
-        private String salesAcc;
-        private String audiCnt;
-        private String audiInten;
-        private String audiChange;
-        private String audiAcc;
-        private String scrnCnt;
-        private String showCnt;
+    public static class MovieItem implements ApiResponse {
+        private final static String CATEGORY = "movie";
+
+        @JsonProperty("id")
+        private Integer id;
+        @JsonProperty("original_title")
+        private String originalTitle;
+        @JsonProperty("poster_path")
+        private String posterPath;
+        @JsonProperty("release_date")
+        private String releaseDate;
+        @JsonProperty("title")
+        private String title;
+        @JsonProperty("overview")
+        private String overview;
+
+        @JsonProperty("poster_path")
+        public void setPosterPath(String posterPath) {
+            this.posterPath = "https://image.tmdb.org/t/p/w300" + posterPath;
+        }
+
+        @Override
+        public ZonedDateTime getEndAt() {
+            return null;
+        }
+
+        @Override
+        public String getAddress() {
+            return null;
+        }
+
+        @Override
+        public String getHashString() {
+            return HashCodeGenerator.hashString(getCategory(), String.valueOf(getId()));
+        }
+
+        @Override
+        public String getCategory() {
+            return CATEGORY;
+        }
+
+        @Override
+        public CatcherItem toEntity(Location location, Category category) {
+            DateTimeFormatter dateFormatter = DateTimeFormatter.ISO_LOCAL_DATE;
+            LocalDate localDate = LocalDate.parse(releaseDate, dateFormatter);
+            ZonedDateTime startAt = localDate.atStartOfDay(ZoneId.of("Asia/Seoul"));
```

Comment:
```suggestion
            ZonedDateTime startAt = localDate.atStartOfDay(zoneId);
```

ApiResponse 인터페이스 내부에 zoneId 변수있는데, 사용해도 좋을거같아용